File: src/test/java/Fo76String.java
import com.fasterxml.jackson.annotation.JsonProperty;

public class Fo76String {

  @JsonProperty("EDID")
  private String edid;
  @JsonProperty("REC")
  private String rec;
  @JsonProperty("Source")
  private String source;
  @JsonProperty("Dest")
  private String dest;
  @JsonProperty("-List")
  private String list;
  @JsonProperty("-sID")
  private String sid;

  public String getList() {
    return list;
  }

  public void setList(String list) {
    this.list = list;
  }

  public String getSid() {
    return sid;
  }

  public void setSid(String sid) {
    this.sid = sid;
  }

  public String getEdid() {
    return edid;
  }

  public void setEdid(String edid) {
    this.edid = edid;
  }

  public String getRec() {
    return rec;
  }

  public void setRec(String rec) {
    this.rec = rec;
  }

  public String getSource() {
    return source;
  }

  public void setSource(String source) {
    this.source = source;
  }

  public String getDest() {
    return dest;
  }

  public void setDest(String dest) {
    this.dest = dest;
  }
}

File: src/main/java/com/manson/fo76/domain/Fo76String.kt
package com.manson.fo76.domain

import com.fasterxml.jackson.annotation.JsonProperty

class Fo76String {
    @JsonProperty("EDID")
    var edid: String = ""

    @JsonProperty("REC")
    var rec: Any? = null

    @JsonProperty("Source")
    var source: String? = null

    @JsonProperty("Dest")
    var dest: String? = null

    @JsonProperty("-List")
    var list: String? = null

    @JsonProperty("-sID")
    var sid: String? = null
    var lang: String = "en"
}